db/migrate/20180214191046_create_shifts.rb

```diff
@@ -0,0 +1,10 @@
+class CreateShifts < ActiveRecord::Migration[5.1]
+  def change
+    create_table :shifts do |t|
+      t.datetime :start_time
+      t.datetime :end_time
```

Comment:
I don't see a user_id in here -- if shifts do belong to users, you may want to add one.